### PR TITLE
document some existing unstable features

### PR DESCRIPTION
**Pull request by japaric**
"msp430-interrupt", "ptx-kernel" and #![compiler_builtins_lib]

r? @steveklabnik

### CODE REVIEW

File: src/doc/unstable-book/src/abi-ptx.md [hunk 1]
```diff
@@ -1,5 +1,59 @@
 # `abi_ptx`
 
-The tracking issue for this feature is: None.
+The tracking issue for this feature
+is: [38788](https://github.com/rust-lang/rust/issues/38788)
```
Comment (steveklabnik) [inline src/doc/unstable-book/src/abi-ptx.md]: we have a standard form for this stuff, like this: https://github.com/rust-lang/rust/pull/41135/files#diff-d6ab7d3e772d67c4944af27d29e229cdR3

mind changing this to be identical? Thanks 😄

**Review by steveklabnik — CHANGES_REQUESTED** [review]
one teeny tiny formatting nit, and then r=me. Thanks so much!

File: src/doc/unstable-book/src/abi-ptx.md (hunk 1 above)
Comment (japaric) [inline src/doc/unstable-book/src/abi-ptx.md]: Sure thing. 😄

**Comment by TimNN** [comment]
@bors retry

* appveyor already failed (unrelated to this, fix in #41147)

**Comment by TimNN** [comment]
@bors retry

* #41147

**Comment by TimNN** [comment]
@bors retry

* #41147

**Pull request merged**